ETL.md

```diff
@@ -35,12 +35,20 @@ For more information, see the [`kiba`] `README.md` file.
 
 ## Usage
 
-Then run the rake task to import the records,
+Then run the rake tasks to import the records,
 transform them,
 and load them into the database:
 
 ```
-bundle exec rake etl:import_local_epub
```

Comment:
I removed the line because JSON importing can be used for data that originated from EPUBs, so doing this import local epub will be unnecessary.  I'd argue we should just be loading the JSON representation of EPUBs since the EPUB file alone doesn't provide all the info we want -- important metadata, including the book's summary (metadata.abstract field).  That ONIX metadata is already present in the JSON files here: s3://knowledge.commons/valid_data/resources/island_press/

Some extra wins are that the JSON ingestion is faster and the specific implementation of EPUB parsing in ruby chokes on a number of valid files, which doesn't happen for the EPUB parsing to JSON I'm using.